resource-timing/resource-timing-level1.sub.html

```diff
@@ -14,6 +14,13 @@
 <body>
   <div id="log"></div>
   <pre id="output"></pre>
-  <script src="resource-timing.js"></script>
+  <script>
+      if (window.location.hostname != "{{domains[www]}}") {
```

Comment:
This isn't allowed. You can't reload the test page because the harness doesn't support that. If you want to be loaded on a subdomain you need to window.open a new window or something.

Reply:
Can I `window.open()` to the same page?

Reply:
post-holidays friendly ping